File: 2026POTD/JANUARY/24-01-2026/1877.cs
public class Solution {
    public int MinPairSum(int[] nums) {
        Array.Sort(nums);
        int i = 0, j = nums.Length - 1, ans = 0;
        while (i < j) {
            ans = Math.Max(ans, nums[i] + nums[j]);
            i++;
            j--;
        }
        return ans;
    }
}

File: 2026POTD/JANUARY/24-01-2026/1877.ts
function minPairSum(nums: number[]): number {
    nums.sort((a, b) => a - b);
    let i = 0, j = nums.length - 1, ans = 0;
    while (i < j) {
        ans = Math.max(ans, nums[i] + nums[j]);
        i++;
        j--;
    }
    return ans;
}